### PR TITLE
Ft support test nginx

### PR DESCRIPTION
**Pull request by wklken**
- support test-nginx

### CODE REVIEW

**Comment by wklken** [comment]
depend on this: https://github.com/TencentBlueKing/blueking-apigateway-apisix/pull/10
should handle the conflict after the #10 merged

File: src/apisix/ci/run-test-nginx.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    authapi:
+      addr: "http://authapi.example.com"
+    bkauth-legacy:
+      addr: "http://bkauth-legacy.example.com"
+    bkauth:
+      addr: "http://bkauth.example.com"
+    login-tencent:
+      addr: "http://login-tencent.example.com"
+    login:
+      addr: "http://login.example"
```
Comment (piglei) [inline src/apisix/ci/run-test-nginx.sh:58]: 缺少 `.com`

File: src/apisix/tests/test-bk-opentelemetry.lua [hunk 2]
```diff
@@ -130,6 +131,7 @@ describe(
                                 bk_resource_name = RANDSTR(),
                                 bk_service_name = RANDSTR(),
                                 bk_request_id = RANDSTR(),
+                                x_request_id = RANDSTR(),
                                 request_uri = "/hello/?a=1&b=2",
                                 uri = "/hello/"
                             }
```
Comment (piglei) [inline src/apisix/tests/test-bk-opentelemetry.lua:137]: 相关文档中添加对新增的 make 指令的简要说明。

File: src/apisix/ci/run-test-nginx.sh (hunk 1 above)
Comment (wklken) [reply src/apisix/ci/run-test-nginx.sh:58]: done

File: src/apisix/tests/test-bk-opentelemetry.lua (hunk 2 above)
Comment (wklken) [reply src/apisix/tests/test-bk-opentelemetry.lua:137]: done

**Review by zhu327 — APPROVED** [review]

**Pull request closed without merging**